de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5489600_sys_gh3741_RenameColumn.sql

```diff
@@ -0,0 +1,35 @@
+-- 2018-03-28T15:25:02.627
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Column SET AD_Element_ID=1047, ColumnName='Processed', Description='Checkbox sagt aus, ob der Beleg verarbeitet wurde. ', Help='Verarbeitete Belege dürfen in der Regel nich mehr geändert werden.', Name='Verarbeitet',Updated=TO_TIMESTAMP('2018-03-28 15:25:02','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=559089
+;
+
+-- 2018-03-28T15:25:02.630
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Field SET Name='Verarbeitet', Description='Checkbox sagt aus, ob der Beleg verarbeitet wurde. ', Help='Verarbeitete Belege dürfen in der Regel nich mehr geändert werden.' WHERE AD_Column_ID=559089
+;
+
+-- 2018-03-28T15:25:24.166
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Column SET Name='Freigegeben',Updated=TO_TIMESTAMP('2018-03-28 15:25:24','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=559089
+;
+
+-- 2018-03-28T15:25:24.169
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Field SET Name='Freigegeben', Description='Checkbox sagt aus, ob der Beleg verarbeitet wurde. ', Help='Verarbeitete Belege dürfen in der Regel nich mehr geändert werden.' WHERE AD_Column_ID=559089
```

Comment:
i think rather than editing the field, you should set "Abw. Feldbenennung" in AD_Field. that's what i mean when i wrote 
>  name it via AD_Element

..because "Abw. Feldbenennung" references AD_Element. If you just edit AD_Field.Name like this, it can be overwritten the next time the terminologiy is synched

Reply:
done!
Thx!
I wasn't aware of it!